packages/lib-classifier/src/store/RootStore.js

```diff
@@ -83,12 +83,28 @@ const RootStore = types
       self.subjects.reset()
     }
 
+    function _afterInitialize() {
+      // Because the root store is initialized all the sub-stores are as well
+      // We enable listening between this root store's stores by calling an afterRootInitialize() method on all these sub-stores
+      const allKeys = keys(self);
+      allKeys.forEach(key => {
+        // Filter the current model to only the observable properties of the root store since that's all our models
+        const isFunction = typeof self[key] === 'function';
+        const isVolatile = !isObservableProp(self, key);
+        if (isFunction || isVolatile) return;
```

Comment:
[coding style] Semicolons, what are you guys doing here in FEM?

Reply:
99% of FEM does not include semicolons. This is a precedent set before my time 🤷‍♀️  I recommend including a .prettierrc in the folder where you edit FEM files with the following:
```
  "jsxSingleQuote": true,
  "singleQuote": true,
  "semi": false,
  "trailingComma": "none",
  "arrowParens": "avoid"
```